### PR TITLE
Add getAllKeys Method to SyncStorage

### PR DESCRIPTION
**Pull request by iamsoorena**
I thought since I use these methods All the time, It would be a good idea to suggest adding them to this library.

### CODE REVIEW

**Comment by raphaelpor** [comment]
Hello @iamsoorena!

Thanks for your PR. I'll help you with this error. 😉

**Comment by olfek** [comment]
@raphaelpor @iamsoorena I disagree with adding the `push` and `update` functions because the `set` function already does the job of both. I do agree with having a `getAllKeys` function though.

**Comment by raphaelpor** [comment]
@olfek I agree with you.

@iamsoorena Could you please remove the `push` and `update` functions? Let's keep this PR only for `getAllKeys`. 😉

**Comment by iamsoorena** [comment]
@raphaelpor It's done.

File: README.md [hunk 1]
```diff
@@ -93,3 +93,27 @@ SyncStorage.remove('foo')
     console.log(error);
   });
 ```
+
+#### update(key: _string_, value: _Any type_)
```
Comment (raphaelpor) [inline README.md]: I think it would to be `getAllKeys`, right? :)

Comment (iamsoorena) [reply README.md]: Yes, I forgot that.

**Comment by raphaelpor** [comment]
Thanks, guys! ❤️ 
@iamsoorena @olfek

**Pull request merged**